.env.dev

```
@@ -0,0 +1,17 @@
+PYTHONUNBUFFERED=1
+SECRET_KEY=secret
+DATABASE_URL='postgis://docker:docker@db:5432/dtpstat'
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)